Doc/faq/programming.rst

```diff
@@ -114,7 +116,7 @@ Core Language
 Why am I getting an UnboundLocalError when the variable has a value?
 --------------------------------------------------------------------
 
-It can be a surprise to get the UnboundLocalError in previously working
+It can be a surprise to get the :exc:`UnboundLocalError` in previously working
```

Comment:
Here and in other sections, I just do a link at the first mention and leave the rest in the section as literal or formatted as I think linking all of them is a bit visually distracting, but let me know if linking everywhere is preferred.

Reply:
Good point, forgot about that. I'll scan through the file again later and see if I can catch a few more of these.